cmd/dtmate/cmd/fmt.go

```diff
@@ -53,6 +63,7 @@ func listConversionsSpecifiers() {
 		{`%R`, `equivalent to %H:%M`},
 		{`%r`, `equivalent to %I:%M:%S %p`},
 		{`%S`, `the second as a decimal number (00-60)`},
+		{`%s`, `the number of seconds since the Epoch, 1970-01-01 00:00:00 +0000 (UTC)`},
```

Comment:
It's a pity you don't have a format for milliseconds

Your lib suggests %f or %L

https://github.com/lestrrat-go/strftime

> POSIX does not specify how to print out milliseconds, but popular implementations allow %f or %L to achieve this.

Reply:
Agreed.